.github/workflows/dapp-build.yml

```diff
@@ -0,0 +1,82 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow.json
+---
+name: "DApp Build"
+on:
+    workflow_call:
+        inputs:
+            dapp-name:
+                required: true
+                type: string
+        outputs:
+            dapp-image:
+                description: "The DApp built container image"
+                value: ${{ jobs.build_package.outputs.dapp-image }}
+        secrets:
+            DOCKER_ORGANIZATION:
+                required: true
+            DOCKER_USERNAME:
+                required: true
+            DOCKER_PASSWORD:
+                required: true
+
+jobs:
+    build_package:
+        runs-on: ubuntu-22.04
+        outputs:
+            dapp-image: ${{ steps.set-dapp-image.outputs.dapp-image }}
+        steps:
+            - uses: actions/checkout@v3
+
+            # Add support for more platforms with QEMU (necessary to run RISC-V containers)
+            - name: Set up QEMU
+              uses: docker/setup-qemu-action@v2
+
+            - name: Set up Docker Buildx
+              uses: docker/setup-buildx-action@v2
+
+            - name: Docker meta
+              id: meta
+              uses: docker/metadata-action@v4
+              with:
+                  bake-target: server
+                  images: |
+                      name=ghcr.io/${{ github.repository_owner }}/honeypot
+                      name=docker.io/${{ github.repository_owner }}/honeypot,enable=${{ github.repository_owner == 'cartesi' }}
```

Comment:
The `enable` is wrong here.
The repository_owner is now always `cartesi` (except in case of forks, but forks won't run anyway).
The `enable` in this case must exclude PR triggers.

```
enable=${{ github.event_name != 'pull_request' }}
```